acquisitions/src/main/resources/thunderjet/cross-modules/features/pay-invoice-and-delete-piece.feature

```diff
@@ -35,31 +27,31 @@ Feature: Pay an invoice and delete a piece
     * def invoiceId = callonce uuid5
     * def invoiceLineId = callonce uuid6
 
-
-  Scenario: Prepare finances
+    # Prepare finances
     * configure headers = headersAdmin
-    * def v = call createFund { id: #(fundId) }
-    * def v = call createBudget { id: #(budgetId), fundId: #(fundId), allocated: 1000 }
-
+    * callonce createFund { id: #(fundId) }
+    * callonce createBudget { id: #(budgetId), fundId: #(fundId), allocated: 1000 }
+    * configure headers = headersUser
 
-  Scenario: Create an order
+  Scenario: Pay an invoice and delete a piece
+    # 1. Create an order
     * def v = call createOrder { id: #(orderId) }
 
+    # 2. Create an order line
+    * table locations
+      | locationId         | quantity | quantityPhysical |
+      | globalLocationsId  | 2        | 2                |
+    * def v = call createOrderLine { id: '#(poLineId)', orderId: '#(orderId)', fundId: '#(fundId)', quantity: 2, locations: '#(locations)' }
 
-  Scenario: Create an order line
-    * def v = call createOrderLine { id: #(poLineId), orderId: #(orderId), fundId: #(fundId) }
-
-
-  Scenario: Open the order
+    # 3. Open the order
     * def v = call openOrder { orderId: #(orderId) }
 
-
-  Scenario: Receive the piece
+    # 4. Receive the piece
     Given path 'orders/pieces'
     And param query = 'poLineId==' + poLineId
     When method GET
     Then status 200
-    And match $.totalRecords == 1
+    And match $.totalRecords == 2
```

Comment:
I guess we change to 2 to allow 1 for deletion and 1 to remain

Reply:
There is two pieces, currently getting total 2 and deleting one. didn't introduce extra case, just fixed